compiler/src/lnntoamm/Types.ts

```diff
@@ -123,6 +139,7 @@ export class Builtin extends Type {
             ty.methods.every(m => this.compatibleWithConstraint(m, scope)) &&
             ty.operators.every(o => this.compatibleWithConstraint(o, scope));
     } else {
+      console.log(ty);
```

Comment:
Delete the debug log? Or just keep it around since this `TODO` means anything hitting this path is going to hard fail, anyways?

Reply:
I can delete it, but yeah `TODO` does throw an exception so the log is just to give some extra info

Reply:
Nah, just keep it.